gcp/opentelemetry-demo-values.yaml

```diff
@@ -72,12 +89,12 @@ opentelemetry-collector:
     service:
       pipelines:
         logs:
-          processors: [memory_limiter, resourcedetection, resource, batch]
+          processors: [resourcedetection, resource, memory_limiter, batch]
```

Comment:
ideally, the memory limiter is always first (drop quickly when hitting the memory limit)

Reply:
This is what was needed to match the current config: https://github.com/quentinmit/opentelemetry-demo/blob/18110944d5f624e55d5678f5f7c0548786db95bf/kubernetes/opentelemetry-demo.yaml#L306

I can change it but I was trying to stick to what was there.

Reply:
ah, got it.  This is fine, then.